man/systemd.network.xml

```diff
@@ -3042,6 +3042,25 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[QuickFairScheduler] Section Options</title>
+    <para>The <literal>[QuickFairScheduler]</literal> section manages the queueing discipline
+    (qdisc) of Quick Fair Scheduler (QFQ).</para>
+
+    <variablelist class='network-directives'>
+      <xi:include href="tc.xml" xpointer="qdisc-parent" />
+      <xi:include href="tc.xml" xpointer="qdisc-handle" />
+
+      <varlistentry>
+        <term><varname>Weight=</varname></term>
```

Comment:
Weight is a property for the qfq class, not for qdisc.

Reply:
ohh yes . I totally misread it . removed. thanks for the review.